generate_instructions_test.py

```diff
@@ -258,3 +252,16 @@ def test_solve_board_final_action_yields_goal(self):
             assert block in self.goal_config.final_blocks
         for block in self.goal_config.final_blocks:
             assert block in final_config.final_blocks
+
+    def test_solve_board_stress(self):
+        num_blocks = 500
+        letters = string.ascii_uppercase
+        colors = ['Red', 'Green', 'Blue', 'Gold', 'Brown', 'Purple', 'Yellow', 'Pink']
+
+        start_config = gi.random_configuration(num_blocks, letters, colors)
+        final_config = start_config.scatter().mark_complete()
+
+        actions = gi.solve_board(start_config, final_config)
+        solved_config = actions[len(actions) - 1].end_conf
+
+        assert final_config == solved_config
```

Comment:
I'm second guessing this a little considering how long CI is suddenly taking. On the other hand, I don't think we need super low latency for the PR test turnarounds.

Reply:
I'm not sure if this is the problem... The builds seem to have just not started. Maybe their servers are down or something?

Reply:
Yeah I don't think this is actually a problem